modules/ROOT/pages/managing-users.adoc

```diff
@@ -105,6 +106,28 @@ Verify that your identity configuration is correct before creating the identity
 
 If your identity provider supports uploading service-provider metadata, you can click the *Anypoint service provider metadata* link to download your SAML configuration settings in an XML file. You can then upload this file to your identity provider to configure Anypoint Platform. Note that you must have a SAML 2.0 identity configuration to export this data.
 
+
+== SSO Redirects using Cookies
```

Comment:
```suggestion
== SSO Redirects Using Cookies
```

Reply:
I'm assuming that "Redirects" is being used here as a noun and not a verb?

That said, I wonder if it would be more precise to add something to the title that tells what about this thing is contained in the narrative: SSO Redirect Process Using Cookies? How SSO Uses Cookies to Return You to Where You Started (no, not that...but you see where I am going with this...)?

Reply:
```suggestion
== SSO Redirects Process Using Cookies
```